AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/AwsCryptographyKeyStoreAdminOperations.dfy

```diff
@@ -171,15 +204,13 @@ module AwsCryptographyKeyStoreAdminOperations refines AbstractAwsCryptographyKey
   method CreateKey ( config: InternalConfig , input: CreateKeyInput )
     returns (output: Result<CreateKeyOutput, Error>)
   {
-    // return Failure(Types.KeyStoreAdminException(message := "Implement me"));
-    var keyManagerStrat :- resolveKmsStratgety(input.strategy, config);
+    var keyManagerStrat :- ResolveStrategy(input.strategy, config);
     :- Need(
-      // Remove before Prod
       keyManagerStrat.reEncrypt?,
-      Types.KeyStoreAdminException(message :="BETA :: Only Re Encrypt is supported!!")
+      Types.KeyStoreAdminException(message :="Only ReEncrypt is supported at this time.")
     );
 
-    var legacyConfig :- kmsInputsToLegacyConfig(keyManagerStrat, input.kmsArn, config);
+    var legacyConfig :- LegacyConfig(keyManagerStrat, input.kmsArn, config);
 
     assume {:axiom} legacyConfig.kmsClient.Modifies < MutationLie();
```

Comment:
add a comment why you are using this particular axiom

Reply:
resolved in a2a28ea23